pkgs/top-level/aliases.nix

```diff
@@ -409,8 +409,7 @@ mapAliases ({
   slurm-llnl-full = slurm-full; # renamed July 2017
   slurm-full = slurm; # added 2018-05-1
   smbclient = samba; # added 2018-04-25
-  slim = throw "slim has been removed. Please use a different display-manager"; # added 2019-11-11
-  slimThemes = throw "slimThemes has been removed because slim has been also"; # added 2019-11-11
+  slim = slim-ng; # added 2019-11-11
   net_snmp = net-snmp; # added 2019-12-21
```

Comment:
This isn't correct. An alias is supposed to be a bind to something that's equivalent, and we've already addressed that it isn't anymore.

```
slim = throw "slim has been removed. Please use the replacement slim-ng or a different display-manager"
```